grails-app/assets/javascripts/project-activity-index.js

```diff
@@ -0,0 +1,99 @@
+// todo: Delete?
```

Comment:
What needs to be deleted/refactored here?

Reply:
sorry for the delay in addressing this. it has been deleted.